db/migrate/20160302192430_add_devise_to_users.rb

```diff
@@ -0,0 +1,49 @@
+class AddDeviseToUsers < ActiveRecord::Migration
+  def self.up
+    change_table(:users) do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      t.integer  :sign_in_count, default: 0, null: false
+      t.datetime :current_sign_in_at
+      t.datetime :last_sign_in_at
+      t.string   :current_sign_in_ip
+      t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
```

Comment:
Since this is how devise acts when running the `rails g devise MODEL` command I thought it would make sense to leave it 'as-is' so it's easier for Learn Experts to compare to what students are creating. (I'm assuming here that students won't be going through and removing the comments).